### DISCUSSION TITLE
Verification of Path Loss Calculation Steps Using CIR Values

### DISCUSSION THREAD

**Discussion by alialsahlany** [NVlabs/sionna#560] [Q&A]
![image](https://github.com/user-attachments/assets/42d33889-0217-4f46-b92c-3264eef46068)

**Comment by jhoydis** [comment]
Hi @alialsahlany,

You would typically use the (2). However, the path loss would be -(4). Without the minus sign, you compute the path gain.

**Reply by alialsahlany** [reply]
> Hi @alialsahlany,
> 
> You would typically use the (2). However, the path loss would be -(4). Without the minus sign, you compute the path gain.

Thank you for your response Mr. Jakob @jhoydis .
So, the mathematical model for calculating path loss will be as indicated below.
![image](https://github.com/user-attachments/assets/a3b02a5d-fffd-4281-8b58-bb4f1934b3b8)
I have another question: How can I determine or control the transmitter gain, receiver gain, and transmitted power?

**Reply by jhoydis** [reply]
I am not sure what you mean by transmitter and receiver gain in this context. The pathloss already takes into account the TX and RX antennas which have a certain gain ([see](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.antenna.compute_gain)). 

The pathloss is independent of the transmitted power. When you do link-level simulations, you can either scale the transmitted signal to have a certain power or scale the noise power. In both cases, the only metric you care about is the signal-to-noise ratio.

If you wanted to compute the received signal strength, it would be RX Power (dB) = TX Power (dB) - Pathloss (dB).

**Reply by Masoudsadeghian** [reply]
Hi @jhoydis,

I have a related question: Why is the total CIR calculated as the sum of the squared amplitudes of each path coefficient $a_i$, rather than the squared amplitude of the sum of the path coefficients $a_i$?

I noticed the same approach is used in calculating the coverage map, as written in the Sionna documentation, from equation (52) here [.](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.coverage_map).

If I’m correct in understanding that the total CIR in the coverage map section is based on the sum of the squared amplitudes of each path coefficient $a_i$, how does precoding impact this in multi-antenna scenarios?

Thanks in advance.
